spec/menu_spec.rb

```diff
@@ -0,0 +1,11 @@
+
+require'menu'
+describe Menu do
+
+  subject(:menu) { described_class.new }
+  # let(:menu) {double :menu}
+
+    it 'displays menu list' do
+      expect(menu.show_menu).to include('chips' => 2.00)
+    end
+  end
```

Comment:
Lint/BlockAlignment: `end` at 11, 2 is not aligned with `describe Menu do` at 3, 0.